edx_proctoring/api.py

```diff
@@ -649,6 +648,11 @@ def get_exam_attempt_data(exam_id, attempt_id, is_learning_mfe=False):
         ),
     }
 
+    if attempt['status'] == ProctoredExamStudentAttemptStatus.ready_to_submit:
+        if (can_continue := _does_time_remain(attempt)) and exam['is_proctored']:
+            can_continue = provider and not provider.should_block_access_to_exam_material()
+        attempt_data['can_continue'] = can_continue
```

Comment:
This means the attribute will sometimes be present in the dictionary, and in other attempt status, it won't be present. Is this OK? Will this be handled properly on the frontend?